src/ProjectNeon/Assets/VFX/RandomPlayAnimation.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using System.Collections;
+
+public class RandomPlayAnimation : MonoBehaviour {
+ 
+	public void Start()
+    { 
+		this.gameObject.GetComponent<SpriteRenderer> ().enabled = false;
+		StartCoroutine(Wait());
+	}     
+
+	IEnumerator  Wait()
+    {
+		yield return new WaitForSeconds(Random.Range(5, 20)); //for waiting
+		//InvokeRepeating("OnOf", Random.Range(10, 20), Random.Range(10, 20)); //for showing (time-repeat)
+		OnOf();
+	}
+	
+	public void OnOf()
```

Comment:
Usually we call this `Toggle`. If you want to use the words "On" and "Off" then it should be the whole word "Off"